src/components/UI/Text/Textarea.vue

```diff
@@ -194,10 +204,12 @@ export default {
     },
   },
   methods: {
-    inputChange(event) {
+    update() {
+      this.$emit("update:value", this.editor.getText() ? this.editor.root.innerHTML : "");
```

Comment:
Is there any case where `this.editor.getText()` will return a false boolean? I mean, is there any case where this if/else condition will return ""?

Reply:
I cannot think of any case right now.But  I kept this to avoid edge cases if there are any,should I change it then?